tonic/src/server/mod.rs

```diff
@@ -15,3 +15,12 @@ pub use self::grpc::Grpc;
 pub use self::service::{
     ClientStreamingService, ServerStreamingService, StreamingService, UnaryService,
 };
+
+/// A trait to provide a static reference to the service's
+/// name. This is used for routing service's within the router.
+pub trait NamedService {
+    /// The `Service-Name` as described [here].
+    ///
+    /// [here]: https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#requests
+    const NAME: &'static str;
```

Comment:
Do we want to change this to a method that returns a &'static str? OR keep it as a const?

Reply:
Isn't this also a breaking change since its moving modules?

Reply:
Ah its not :)

Reply:
yeah, it's re-exported in `transport`.